### ISSUE TITLE
import error on archinfo 8.18.10.25

### ISSUE THREAD

**Issue by xorond** [angr/archinfo#51]
```
Python 3.7.1 (default, Oct 22 2018, 10:41:28) 
[GCC 8.2.1 20180831] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import archfino
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'archfino'
>>> import archinfo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.7/site-packages/archinfo/__init__.py", line 12, in <module>
    from .arch_amd64    import ArchAMD64
  File "/usr/lib/python3.7/site-packages/archinfo/arch_amd64.py", line 252, in <module>
    register_arch([r'.*amd64|.*x64|.*x86_64|.*metapc'], 64, Endness.LE, ArchAMD64)
  File "/usr/lib/python3.7/site-packages/archinfo/arch.py", line 647, in register_arch
    all_arches.append(my_arch(endness))
  File "/usr/lib/python3.7/site-packages/archinfo/arch_amd64.py", line 33, in __init__
    super(ArchAMD64, self).__init__(endness)
  File "/usr/lib/python3.7/site-packages/archinfo/arch.py", line 219, in __init__
    self.register_size_names[(reg.vex_offset + off, sz)] = name
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'
```

**Comment by rhelmot** [comment]
oh my god... I can't believe I broke this again in between making fixing it and making a release. It's fixed again.

**Comment by DimitryRakhlei** [comment]
I seem to have run into the same issue when trying to run angr.
```
Python 3.7.1 (default, Oct 22 2018, 10:41:28) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.1.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import angr                                                                                     
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-2458dfe59970> in <module>
----> 1 import angr

/usr/lib/python3.7/site-packages/angr/__init__.py in <module>
     26 
     27 # this must happen first, prior to initializing analyses
---> 28 from .sim_procedure import SimProcedure
     29 from .procedures import SIM_PROCEDURES, SimProcedures, SIM_LIBRARIES
     30 

/usr/lib/python3.7/site-packages/angr/sim_procedure.py in <module>
      2 import copy
      3 import itertools
----> 4 from cle import Symbol
      5 
      6 import logging

/usr/lib/python3.7/site-packages/cle/__init__.py in <module>
     14 # pylint: disable=wildcard-import
     15 from . import utils
---> 16 from .loader import *
     17 from .memory import *
     18 from .errors import *

/usr/lib/python3.7/site-packages/cle/loader.py in <module>
      6 from collections import OrderedDict
      7 
----> 8 import archinfo
      9 
     10 from .address_translator import AT

/usr/lib/python3.7/site-packages/archinfo/__init__.py in <module>
     10 from .arch import *
     11 from .defines import defines
---> 12 from .arch_amd64    import ArchAMD64
     13 from .arch_x86      import ArchX86
     14 from .arch_arm      import ArchARM, ArchARMEL, ArchARMHF

/usr/lib/python3.7/site-packages/archinfo/arch_amd64.py in <module>
    250 
    251 
--> 252 register_arch([r'.*amd64|.*x64|.*x86_64|.*metapc'], 64, Endness.LE, ArchAMD64)

/usr/lib/python3.7/site-packages/archinfo/arch.py in register_arch(regexes, bits, endness, my_arch)
    645         all_arches.append(my_arch(Endness.LE))
    646     else:
--> 647         all_arches.append(my_arch(endness))
    648 
    649 

/usr/lib/python3.7/site-packages/archinfo/arch_amd64.py in __init__(self, endness)
     31         if endness != Endness.LE:
     32             raise ArchError('Arch AMD64 must be little endian')
---> 33         super(ArchAMD64, self).__init__(endness)
     34         self.argument_register_positions = {
     35             self.registers['rdi'][0]: 0,

/usr/lib/python3.7/site-packages/archinfo/arch.py in __init__(self, endness, instruction_endness)
    217                 if self.name in {'X86', 'AMD64'} and name in {'bp', 'sp', 'ip'}:
    218                     continue
--> 219                 self.register_size_names[(reg.vex_offset + off, sz)] = name
    220 
    221         # Unicorn specific stuff

TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'
```

**Comment by ltfish** [comment]
This has been fixed in the latest GitHub version.

**Comment by Francis-OS** [comment]
Mime is somewhat different. 
Trying run a python27 test script built using cx_Freeze only to get
Traceback (most recent call last) 
File
"C:\Python2.7.2\lib\site-packages\cx_Freeze\initscripts\_startup_.py",
Line 12, in <module>
_import_(name +"_init_")

**Comment by Francis-OS** [comment]
Any help please

**Comment by rhelmot** [comment]
This bug affects a 8 month old version of angr, and the traceback you posted doesn't show anything to do with angr. Please open a new issue with more information if you want help.
